### ISSUE TITLE
Django 1.11 - ValueError at / Missing staticfiles manifest entry for ''

### ISSUE THREAD

**Issue by Adamcina** [evansd/whitenoise#180]
My project consists of three separate apps, every one of them is using whitenoise for static files and AWS S3 for serving media files, and every app uses separate settings (appx_settings.py file). Third app is loading some models from the second app (if that matters somehow). 
I've been running python 2.7.5 Django 1.9.7 for a while and everything was working perfectly. In the meanwhile i've decided to move on Django 1.11. After upgrading, two of three apps are working fine, while the third one gives me headaches - i constantly get HTTP 500 error, with this line - `ValueError at / Missing staticfiles manifest entry for ''. ` I've been experimenting a lot and i've concluded that whitenoise may be the problem, because with these lines included (in my app3_settings.py):
```
STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
STATIC_ROOT = os.path.join(BASE_DIR, 'project/static/app3') 
STATIC_URL = STATIC_HOST + '/static/'
```
i get `ValueError at / Missing staticfiles manifest entry for ''. ` I've also tried to change `STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'` to `STATICFILES_STORAGE = 'django.contrib.staticfiles.storage.ManifestStaticFilesStorage'` but that didnt help.
If i delete those lines and add these instead:
```
STATIC_ROOT = os.path.join(BASE_DIR, 'static')
STATIC_URL = '/static/'
```
everything works fine. 
Normally, i've done `python collectstatic --setings=app3_settings`. Every django/python plugin is updated to the newest version (whitenoise==3.2).
So, long story short - everything works well under Django 1.9 but after upgrading it to 1.11 (and not touching the settings or anything else) i get HTTP 500 error.
If you may say that this type of question is not suitable for this forum, sorry for that, i've asked this on [StackOverflow](https://stackoverflow.com/questions/46479378/valueerror-at-missing-staticfiles-manifest-entry-for) 6 months ago, but unfortunately didn't get any answer, so i wanted to ask here if someone knows anything about this issue. Thanks in advance!

**Comment by evansd** [comment]
Thanks for the detailed explanation. I've answered your original stackoverflow question as that's probably a better forum for this kind of question:
https://stackoverflow.com/a/49656053/79992
